src/RefinedAst.hs

```diff
@@ -117,6 +117,11 @@ data Exp t where
   LitInt :: Integer -> Exp Integer
   IntVar :: Id -> Exp Integer
   IntEnv :: EthEnv -> Exp Integer
+  -- bounds
+  IntMin :: Int -> Exp Integer
+  IntMax :: Int -> Exp Integer
+  UIntMin :: Int -> Exp Integer
+  UIntMax :: Int -> Exp Integer
```

Comment:
it's helpful for the coq backend to abstract the integer bounds

Reply:
@MrChico @xwvvvvwx can you review the related fixes in https://github.com/ethereum/act/pull/44/commits/2da2812d3ee84fe8518a3f94be713fd28bcbb45b and https://github.com/ethereum/act/pull/44/commits/5122289a6e5ff60f58ea2ce1859863f30435d5a5?

Reply:
those changes look good to me